### PR TITLE
Add DWPD to Hardware Overview dashboard

### PR DESCRIPTION
**Pull request by technowhizz**
Adds a panel in the Hardware Overview dashboard to show DWPD (Drive writes per day) for NVMEs. This is calculated by dividing the total bytes written in the past 24 hours by the drive capacity.

### CODE REVIEW

**Review by oneswig — DISMISSED** [review]
Good to see this

**Review by dougszumski — CHANGES_REQUESTED** [review]
We agreed to add the following before finishing this task:

- Short documentation section to be added explaining the limitations (only nvme supported) and how to customise the dashboard to remove the plot on systems without supported drives.
- Alert to be added, as noted in https://stackhpc.atlassian.net/browse/INFRA-395

**Comment by Alex-Welsh** [comment]
@technowhizz is this still WIP? would it be better to convert to draft for now?

**Comment by technowhizz** [comment]
@Alex-Welsh Nope, its waiting to be merged :)

**Comment by Alex-Welsh** [comment]
> @Alex-Welsh Nope, its waiting to be merged :)

Might want to poke some potential reviewers then

File: releasenotes/notes/dwpd-6b9fb0c8d6d3a570.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    Adds a panel in the Hardware Overview dashboard to show DWPD (Drive writes
+    per day) for NVMEs. This is calculated by dividing the total bytes written
```
Comment (dougszumski) [inline releasenotes/notes/dwpd-6b9fb0c8d6d3a570.yaml:5]: nit: You could include the alert here.

**Review by dougszumski — DISMISSED** [review]
Thanks @technowhizz.

File: releasenotes/notes/dwpd-6b9fb0c8d6d3a570.yaml (hunk 1 above)
Comment (technowhizz) [reply releasenotes/notes/dwpd-6b9fb0c8d6d3a570.yaml:5]: Done

**Comment by technowhizz** [comment]
@oneswig bumped up the alert durations as discussed

**Review by dougszumski — APPROVED** [review]

**Pull request merged**